docs/reference/openapi-rules.md

```diff
@@ -0,0 +1,3 @@
+## OpenAPI Rules
+
+Coming soon...
```

Comment:
Why not skip it for now then?

Reply:
Following PR will cover this shortly. I'd rather tease it than have people think we just forgot to cover something important. It'll be soon, its in this sprint.